docs/using-easybuild.md

```diff
@@ -2,6 +2,8 @@
 
 ## All available easyconfig parameters
 
+## Enabling dependency resolution
+
```

Comment:
Is this change meant to be included in the PR?

Reply:
Yes, because there's a link to this anchor in `what-is-easybuild.md`.

Reply:
Ah yes!